openshift/template.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: fabric8-analytics-bigquery-manifests-job
+metadata:
+  name: fabric8-analytics-bigquery-manifests-job
+  annotations:
+    description: fabric8-analytics-bigquery-manifests-job
+objects:
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    name: fabric8-analytics-bigquery-manifests-job
+    annotations:
+      description: fabric8-analytics-bigquery-manifests-job
+  spec:
+    successfulJobsHistoryLimit: 4
+    failedJobsHistoryLimit: 0
+    concurrencyPolicy: "Forbid"
+    schedule: "${CRON_SCHEDULE}"
+    jobTemplate:
+      spec:
+        template:
+          spec: 
+            restartPolicy: Never
+            containers:
+            - name: bigquery-manifests
+              image: "djsarkar/bigquery-manifests:latest"
+              env:
+                - name: DEBUG
+                  value: "true"
+                - name: AWS_S3_ACCESS_KEY_ID
+                  valueFrom:
+                    secretKeyRef:
+                      key: s3-access-key-id
+                      name: aws
+                - name: AWS_S3_SECRET_ACCESS_KEY
+                  valueFrom:
+                    secretKeyRef:
+                      key: s3-secret-access-key
+                      name: aws
+                - name: GOOGLE_APPLICATION_CREDENTIALS
+                  valueFrom:
+                    secretKeyRef:
+                      key: bq-key
+                      name: bigquery-access
+              resources:
+                requests:
+                  cpu: ${CPU_REQUEST}
+                  memory: ${MEMORY_REQUEST}
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
+parameters:
+  - description: Schedule
+    displayName: Schedule
+    required: true
+    name: CRON_SCHEDULE
+    value: "*/10 * * * *"
+
+  - description: CPU request
+    displayName: CPU request
+    required: true
+    name: CPU_REQUEST
+    value: "500m"
+
+  - description: CPU limit
+    displayName: CPU limit
+    required: true
+    name: CPU_LIMIT
+    value: "800m"
+
+  - description: Memory request
+    displayName: Memory request
+    required: true
+    name: MEMORY_REQUEST
+    value: "512Mi"
+
+  - description: Memory limit
+    displayName: Memory limit
+    required: true
+    name: MEMORY_LIMIT
+    value: "1024Mi" 
```

Comment:
@dipanjanS please see that the CPU and Memory request and limit values are set as per the inferred values during the spike.

Reply:
@samuzzal-choudhury sure I will verify this once more